services/cognitiveservices/v1.0/videosearch/videos.go

```diff
@@ -45,145 +45,148 @@ func NewVideosClientWithBaseURI(baseURI string) VideosClient {
-// whether display strings contain decoration markers such as hit highlighting characters. If true, the strings may
-// include markers. The default is false. To specify whether to use Unicode characters or HTML tags as the markers, see
-// the [textFormat](https://docs.microsoft.com/en-us/rest/api/cognitiveservices/bing-video-api-v7-reference#textformat)
+// header are mutually exclusive; do not specify both. A user interface string is a string that's used as a label
+// in a user interface. There are few user interface strings in the JSON response objects. Also, any links to
+// Bing.com properties in the response objects apply the specified language. textDecorations is a Boolean value
+// that determines whether display strings contain decoration markers such as hit highlighting characters. If true,
+// the strings may include markers. The default is false. To specify whether to use Unicode characters or HTML tags
+// as the markers, see the
+// [textFormat](https://docs.microsoft.com/en-us/rest/api/cognitiveservices/bing-video-api-v7-reference#textformat)
 // query parameter. For information about hit highlighting, see [Hit
-// Highlighting](https://docs.microsoft.com/azure/cognitive-services/bing-news-search/hit-highlighting). textFormat is
-// the type of markers to use for text decorations (see the textDecorations query parameter). Possible values are
-// Raw—Use Unicode characters to mark content that needs special formatting. The Unicode characters are in the range
-// E000 through E019. For example, Bing uses E000 and E001 to mark the beginning and end of query terms for hit
-// highlighting. HTML—Use HTML tags to mark content that needs special formatting. For example, use <b> tags to
+// Highlighting](https://docs.microsoft.com/azure/cognitive-services/bing-news-search/hit-highlighting). textFormat
+// is the type of markers to use for text decorations (see the textDecorations query parameter). Possible values
+// are Raw—Use Unicode characters to mark content that needs special formatting. The Unicode characters are in the
+// range E000 through E019. For example, Bing uses E000 and E001 to mark the beginning and end of query terms for
+// hit highlighting. HTML—Use HTML tags to mark content that needs special formatting. For example, use <b> tags to
 // highlight query terms in display strings. The default is Raw. For display strings that contain escapable HTML
 // characters such as <, >, and &, if textFormat is set to HTML, Bing escapes the characters as appropriate (for
 // example, < is escaped to &lt;).
-func (client VideosClient) Details(ctx context.Context, xBingApisSDK string, query string, acceptLanguage string, userAgent string, clientID string, clientIP string, location string, countryCode string, ID string, modules []VideoInsightModule, market string, resolution VideoResolution, safeSearch SafeSearch, setLang string, textDecorations *bool, textFormat TextFormat) (result VideoDetails, err error) {
```

Comment:
Technically this is a breaking change, we change the method signature here.

Reply:
We will have to exclude this package from the update then.  :(  Are there others?

Reply:
all of the CS ones will have this

Reply:
And the monitor/mgnt/2017-05-01-preview/insights will have breaking changes :(

Reply:
pretty much we cannot modify anything that is already in. We can just leave the ones added ( a new batch dataplane api version and a a new api version for Monitor )